setup/macos_setup.sh

```diff
@@ -0,0 +1,5 @@
+sh setup_macos/macos_dependencies.sh
```

Comment:
I was just copying what was already in the scripts. Do you have a suggested way of executing the other scripts?